app/assets/javascripts/app/views/gallery_view.js

```diff
@@ -0,0 +1,44 @@
+app.views.Gallery = app.views.Base.extend({
+  pictures: function(){
+    return this.$el.find("a.gallery-picture");
+  },
+
+  initialize: function(){
+    var self = this;
+    console.log(this.$el.find("a.gallery-picture").length + " pictures found");
+
+    this.pictures().click(function(event){
+      event = event || window.event;
+      var target = event.target || event.srcElement;
+      var link = target.src ? target.parentNode : target;
+      var links = self.pictures();
+      blueimp.Gallery(links, self.options(event, link));
```

Comment:
Make sure to add it to the predefs in our `.jshintrc`, though it won't take effect for the PR.